torch_xla/csrc/aten_xla_type.h

```diff
@@ -239,12 +239,6 @@ class AtenXlaType {
 
   static at::Tensor& ceil_(at::Tensor& self);
 
-  static at::Tensor celu(const at::Tensor& self, at::Scalar alpha);
```

Comment:
Same question ... XLATensor methods need to stay?

Reply:
There's no XLATensor::celu ;) We're calling back to at::native in these cases.